### ISSUE TITLE
Thank you & Issue

### ISSUE THREAD

**Issue by mstaack** [tuscanicz/BeSimpleSoap#5]
Thank you for your work on making soap easier in php.

its working almost perfectly for me, but if the id is not found/does not pass validation... 
you call `getContentType` on null...

running this:
```
        $soapClientBuilder = new SoapClientBuilder;
        $soapClient        = $soapClientBuilder->build(
            SoapClientOptionsBuilder::createWithDefaults(),
            SoapOptionsBuilder::createWithDefaults('server.de/service?wsdl')
        );
        $soapResponse      = $soapClient->soapCall('findWorld', [['id' => $id]]);
```

throws
```
Call to a member function getContentType() on null in /home/mstaack/server/apache/api.company.de/v2.0/vendor/tuscanicz/soap/src/BeSimple/SoapClient/SoapClient.php on line 333
```

**Comment by tuscanicz** [comment]
Thank you @mstaack, it is definitely a bug. I will check it out later today and fix it.

**Comment by tuscanicz** [comment]
The problem is now resolved, I am gonna publish a fix within a few hours.

To avoid bug with this version and previous, turn on tracing in SoapClientOptions.

The SoapClientOptionsBuilder::createWithDefaults method returns options with SoapClientOptions::SOAP_CLIENT_TRACE_OFF, that is causing the bug.

**Comment by mstaack** [comment]
@tuscanicz  yeah i did that too.....thanks!

**Comment by tuscanicz** [comment]
Fixed in commit b650254. Please, update the library to v4.4 and retest.

**Comment by mstaack** [comment]
awesome
